deploy/ndm-operator.yaml

```diff
@@ -743,4 +745,42 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: METRICS_LISTEN_PORT
+            value: :9101
+---
+# Create NDM cluster exporter service
```

Comment:
The below service also needs to be added to `deploy/yamls`, and then the ndm-operator.yaml should be autogenerated using `make manifests`

Reply:
Okay.